### PR TITLE
first version of Entities with fixtures and unit tests

### PR DESCRIPTION
**Pull request by samirdev2019**
I created the **first version** of the **entities**, they did not specify in detail their **properties** in the context of the projects, for **the fixtures**, I tried to make simple without use of **Faker**,
 the **unit tests** I did them for the getters 

> i'm not sure for the customer properties, specialty username and password; maybe I'll change them when setting up authentication

### CODE REVIEW

File: src/DataFixtures/AppFixtures.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+use Doctrine\Common\Persistence\ObjectManager;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        
+        for ($i=1; $i <= 3; $i++) {
+            $customer = new Customer();
+            $customer->setFullname("free $i");
```
Comment (JeremyGreaux) [inline src/DataFixtures/AppFixtures.php:17]: Tu as ajouté faker, mais tu ne l'utilise pas

File: src/Entity/Customer.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $username;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $password;
+    //, cascade={"persist"}
```
Comment (JeremyGreaux) [inline src/Entity/Customer.php]: A enlever

File: src/Repository/CustomerRepository.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Customer::class);
+    }
+
+    // /**
+    //  * @return Customer[] Returns an array of Customer objects
+    //  */
+    /*
+    public function findByExampleField($value)
```
Comment (JeremyGreaux) [inline src/Repository/CustomerRepository.php]: A supprimer

File: src/Repository/UserRepository.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, User::class);
+    }
+
+    // /**
+    //  * @return User[] Returns an array of User objects
+    //  */
+    /*
+    public function findByExampleField($value)
```
Comment (JeremyGreaux) [inline src/Repository/UserRepository.php]: A supprimer

File: src/DataFixtures/AppFixtures.php (hunk 1 above)
Comment (samirdev2019) [reply src/DataFixtures/AppFixtures.php:17]: oui, j'avais un bug de namespace pour la class Faker, , malgré je l'ai utilisé normalement dans le projet 06 et ça marche bien  \Faker\Factory::create() 
j'avais essayé les deux : Faker\Factory::create()     \Faker\Factory::create()  mais ça bug

**Review by JeremyGreaux — APPROVED** [review]

**Pull request merged**